draft-ietf-scitt-architecture.md

```diff
@@ -354,19 +354,25 @@ The terms "header", "payload", and "to-be-signed bytes" are defined in {{-COSE}}
 
 The term "claim" is defined in {{RFC8392}}.
 
-Append-only Log:
+Statement Sequence:
 
 A sequence of Signed Statements captured by a Verifiable Data Structure.
 
 : see Verifiable Data Structure
 
+Append-only Log:
+
+A Statement Sequence comprising the entire registration history of the Transparency Service.
+
+To make append-only verifiable and transparent, the Transparency Service defines how Signed Statements are made available to Auditors.
```

Comment:
```suggestion
To make the append-only property verifiable, the Transparency Service defines how Signed Statements are made available to Auditors.
```

I am not sure how to parse the original sentence, two things that are easier for me to understand are "append-only is a property of a sequence, we